ui/package.json

```diff
@@ -22,6 +22,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
+  "proxy": "https://cloud-api-dev.calyptia.com",
```

Comment:
should this be prod?

Reply:
This proxy is just for the development server of CRA (create react app).